README.md

```diff
@@ -38,6 +38,8 @@ If you use this bindings from the git repo, be sure to use
   "js-post-build": {
     "cmd": "./node_modules/bs-react-native-monorepo/git-monorepo-usage-trick"
   }
+  // if you this tricky script doesn't work for you correctly, try
```

Comment:
if you this tricky script
-> if this tricky script

(Why wouldn't it work BTW?)

Reply:
Windows or any system that doesn't have vim/ex (I think circle-ci doesn't)

Reply:
I fixed the thing. Thanks :)